packages/cli/src/cmds/validator/handler.ts

```diff
@@ -1,17 +1,18 @@
 import {AbortController} from "@chainsafe/abort-controller";
-import {getClient} from "@chainsafe/lodestar-api";
 import {LevelDbController} from "@chainsafe/lodestar-db";
 import {SignerType, Signer, SlashingProtection, Validator} from "@chainsafe/lodestar-validator";
+import {getMetrics, MetricsRegister} from "@chainsafe/lodestar-validator";
 import {KeymanagerServer, KeymanagerApi} from "@chainsafe/lodestar-keymanager-server";
+import {RegistryMetricCreator, collectNodeJSMetrics, HttpMetricsServer} from "@chainsafe/lodestar";
```

Comment:
might be nice to pull this and the `IGauge` interface at some point?

Reply:
Yeah.. but probably not worth the extra package or dependency relations

Reply:
utilz

Reply:
urrg the recursive dependencies between lodestar/utils and ssz have been awful. Not looking forward to something similar